tests/test_config_flow.py

```diff
@@ -384,59 +379,3 @@ async def test_options_flow_with_zwavejs(
 
         await hass.async_block_till_done()
         assert entry.data.copy() == data
-
-
-async def test_get_entities(hass, lock_data):
-    """Test function that returns entities by domain."""
-    await setup_ozw(hass, fixture=lock_data)
-
-    # Make sure the lock loaded
-    state = hass.states.get("lock.smartcode_10_touchpad_electronic_deadbolt_locked")
```

Comment:
Can we change this to load the zwavejs entities and verify the lock loads instead?

```python
    # Load ZwaveJS
    node = lock_kwikset_910
    state = hass.states.get(KWIKSET_910_LOCK_ENTITY)
```